.circleci/config.yml

```diff
@@ -1375,3 +1406,50 @@ workflows:
       equal: [bump, << pipeline.parameters.action >>]
     jobs:
       - release-train
+
+  # to trigger tests manually, log into circleCI, select the project, a branch, and then click "Trigger Pipeline"
+  # and add a parameter called "run-manual-tests"
```

Comment:
will add a github comment trigger for this as a follow-up PR

Reply:
Not sure how you were planning to implement that but I was thinking maybe we could have something in our CircleCI orb that allows us to establish certain comment to workflow triggers, that way we can reuse it in the other repos in the future as needed. Not needed for this PR of course